src/compiler/checker.ts

```diff
@@ -1100,6 +1103,7 @@ import {
 } from "./_namespaces/ts";
 import * as moduleSpecifiers from "./_namespaces/ts.moduleSpecifiers";
 import * as performance from "./_namespaces/ts.performance";
+import { createSyntacticTypeNodeBuilder } from "./expressionToTypeNode";

```

Comment:
It's probably fine to start doing this if it works, but we generally aren't yet using direct imports and instead export stuff via the big module.

Reply:
Should I change it to the namespace import?

Reply:
I would, yes, just for consistency (and maybe for the people who aren't afraid to use TS internals)